python/tvm/relax/transform/legalize_ops.py

```diff
@@ -824,6 +824,8 @@ def __init__(self, mod: IRModule, customize_legalize_map: Dict[str, LegalizeFunc
                     self.legalize_map[name] = func
 
             def _convert_op(self, call: Call) -> Expr:
+                if not isinstance(call.op, tir.op.Op):
+                    return call
```

Comment:
Can we do this in `visit_call`? This may be better since we can say in `_convert_op` the input call always has op an OpNode.